### ISSUE TITLE
[Sim] track 'log loss' model metric in logs

### ISSUE THREAD

**Issue by trentmc** [oceanprotocol/pdr-backend#965]
### TODOs / DoD

- [ ] Add the metric side-by-side with accuracy, f1, etc in logs

Follow-up issue:

- [#967]( "[Sim, plots](https://github.com/oceanprotocol/pdr-backend/issues/967) "_Plot_ log loss metric"

### On implementing

- [sklearn log_loss](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.log_loss.html)

### Background / motivation

Basic motivation: to account for prediction confidence. This is meaningful given that amount staked is a function of prediction confidence.

Elaboration from "Advances in Financial Machine Learning":

<img src="https://github.com/oceanprotocol/pdr-backend/assets/5305452/978bbef8-aba0-489b-8562-8fa07f371fb3" width="200">
<img src="https://github.com/oceanprotocol/pdr-backend/assets/5305452/94b4575f-3a73-4198-bc81-3dbd978e5732" width="200">
<img src="https://github.com/oceanprotocol/pdr-backend/assets/5305452/aee3864f-272d-4cbf-919f-12d61deea839" width="200">
<img src="https://github.com/oceanprotocol/pdr-backend/assets/5305452/2242c6bd-698c-437f-b7a3-eb29b63922d1" width="200">
<img src="https://github.com/oceanprotocol/pdr-backend/assets/5305452/4810b0ab-25da-49dc-aa9e-af6f6a954f94" width="200">
<img src="" width="200">
<img src="https://github.com/oceanprotocol/pdr-backend/assets/5305452/4367ab6a-dece-4b24-a24d-fe09ad364e30" width="200">
<img src="https://github.com/oceanprotocol/pdr-backend/assets/5305452/9c1ce763-1682-45e1-9835-ae7b1f65905b" width="200">
<img src="" width="200">
<img src="" width="200">
